### PR TITLE
add: usedInVMs for ClusterVirtualMachineImage, VirtualMachineImage, V…

### PR DESCRIPTION
**Pull request by universal-itengineer**
Information about which VMs they are connected to has been added to the status of these resources.

### CODE REVIEW

File: api/clustervirtualmachineimage.yaml [hunk 1]
```diff
@@ -341,6 +341,13 @@ spec:
                   type: string
                   description: |
                     Detailed description of the error. human-readable
+                usedInVMs:
+                  type: array
+                  description: |
+                    A list of virtual machines that use the virtual machine disk
```
Comment (fl64) [inline api/clustervirtualmachineimage.yaml:347]: image

File: api/virtualmachineimage.yaml [hunk 2]
```diff
@@ -363,6 +363,13 @@ spec:
                   type: string
                   description: |
                     Detailed description of the error.
+                usedInVMs:
+                  type: array
+                  description: |
+                    A list of virtual machines that use the disk
```
Comment (fl64) [inline api/virtualmachineimage.yaml:369]: image

**Review by fl64 — COMMENTED** [review]
fit it

File: api/clustervirtualmachineimage.yaml [hunk 3]
```diff
@@ -341,6 +341,13 @@ spec:
                   type: string
                   description: |
                     Detailed description of the error. human-readable
+                usedInVMs:
+                  type: array
+                  description: |
+                    A list of virtual machines that use image
```
Comment (fl64) [inline api/clustervirtualmachineimage.yaml:347]:
```suggestion
                    A list of virtual machines that use the image
```

File: api/virtualmachineimage.yaml [hunk 4]
```diff
@@ -363,6 +363,13 @@ spec:
                   type: string
                   description: |
                     Detailed description of the error.
+                usedInVMs:
+                  type: array
+                  description: |
+                    A list of virtual machines that use image
```
Comment (fl64) [inline api/virtualmachineimage.yaml:369]:
```suggestion
                    A list of virtual machines that use the image
```

**Review by fl64 — CHANGES_REQUESTED** [review]

**Review by distorhead — APPROVED** [review]

**Pull request merged**